WRITING.md

```diff
@@ -49,6 +49,60 @@ As a general guide for writing tone, you can follow the [Google Developers Guide
 
 Also see tips on how to [write inclusive documentation](https://developers.google.com/style/inclusive-documentation) and [write accessible documentation](https://developers.google.com/style/accessibility)
 
+## Recipe Writing
+
+Here are some tips and examples to match Astro Docs' style for instructional content, like a recipe:
+
+### Imperative (instruction, command) tense
+
+Whenever possible, give the reader a direct instruction:
+
+e.g. Type the command...
+
+Do not use _"Let's..."_ or _"Next, we will..."_. You are not sitting there with your reader! (This also means you will never need the words _we_, _us_, and _our_. If you see them, rephrase!)
+
+If you must address the reader, it is OK to use "you" and "your". Especially for emphasis in very important steps where something could go wrong:
```

Comment:
I think here the preposition should be "on"? Feel free to disregard.
```suggestion
If you must address the reader, it is OK to use "you" and "your". Especially for emphasis on very important steps where something could go wrong:
```

Reply:
I think "in a step"... the same way we'd say "Do this in step 3" (At least, that's how I was envisioning the sentence, as being "inside a particular step" while you are writing it as opposed to "putting empasis ON step 3")